sos/report/__init__.py

```diff
@@ -868,24 +868,30 @@ def list_plugins(self):
             _defaults = self.loaded_plugins[0][1].get_default_plugin_opts()
             for _opt in _defaults:
                 opt = _defaults[_opt]
-                val = opt.default
-                if opt.default == -1:
-                    val = TIMEOUT_DEFAULT
+                val = opt.value
+                if opt.value == -1:
+                    if _opt == 'timeout':
+                        val = self.opts.plugin_timeout or TIMEOUT_DEFAULT
+                    elif _opt == 'cmd-timeout':
+                        val = self.opts.cmd_timeout or TIMEOUT_DEFAULT
```

Comment:
Looks better as timeout and cmd-timeout is not respected, however should not be there also postproc? I am not sure why anyone will do it but --no-postproc is still not respected.